<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondRapor</class>
 <widget class="QDialog" name="secondRapor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="ikinci_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tutar_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BEKLENÄ°YOR ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ek_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>EK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
